src/lib.js

```diff
@@ -165,6 +165,7 @@ export default class Migrator {
         this.log('There are no migrations to run'.yellow);
         this.log(`Current Migrations' Statuses: `);
         await this.list();
+        return []
```

Comment:
i'm not sure what you mean. this seems right to me and we've been using it in production. 

the intention is to not error in the case that there are no migrations and we're running in command line mode, right? what am i missing? thanks!